lib/fog/compute/kubevirt/models/server.rb

```diff
@@ -0,0 +1,87 @@
+require 'fog/compute/models/server'
+
+module Fog
+  module Compute
+    class Kubevirt
+      class Server < Fog::Compute::Server
```

Comment:
Why not to extend form [here](https://github.com/fog/fog-kubevirt/blob/master/lib/fog/kubevirt/compute/models/vm.rb) we could inherit most of this code.

Reply:
Since server should be derived from Fog::Compute::Server to inherit its functionalities
We can evaluate changing the VirtualMachine to inherit from Fog::Compute::Server, but then we'll have to mix Vm with VmInstance

Reply:
I am looking for a way to reuse existing code instead of having copy of it.

Reply:
I introduced the VmBase and VmAction modules for that purpose - please let me know what do you think